### PR TITLE
refactor: remove branding style config

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

- Delete branding style config
- Remove slogan config
- Make the logo URL optional
- Change the branding logo display logic:
  - When no logo URL is available, hide the branding logo
  - When only one logo is available, display the only logo
  - When both logo and dark logo are available, display the logo according to the theme

### No Logo

image

### One Logo

image

image

### Two Logo

image

image

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]
LGTM

File: packages/ui/src/utils/logo.ts [hunk 1]
```diff
@@ -16,3 +16,24 @@ export const getLogoUrl = ({ theme, logoUrl, darkLogoUrl, isApple }: GetLogoUrl)
 
   return logoUrl;
 };
+
+type GetBrandingLogoUrl = Omit<GetLogoUrl, 'logoUrl'> & {
+  logoUrl?: Nullable<string>;
+};
+
+export const getBrandingLogoUrl = ({
```
Comment (simeng-li) [inline packages/ui/src/utils/logo.ts:24]: Chould merge with the existing getLogtoUrl method, one single `getBrandingLogtoUrl` should be fine. P.S. do you mind add a UT test for this, thx.

File: packages/ui/src/pages/Consent/index.tsx [hunk 2]
```diff
@@ -14,6 +15,15 @@ const Consent = () => {
...
   const branding = experienceSettings?.branding;
+  const brandingLogo = conditional(
+    branding &&
+      getBrandingLogoUrl({
+        theme,
+        logoUrl: branding.logoUrl,
+        darkLogoUrl: branding.darkLogoUrl,
+      })
+  );
+
```
Comment (simeng-li) [inline packages/ui/src/pages/Consent/index.tsx:26]: move into the getBrandingLogoUrl method?  

```js
const brandingLogo = getBrandingLogoUrl(theme, branding);
```

**Comment by xiaoyijun** [comment]
Convert to draft for a new plan is being drafted.

**Comment by xiaoyijun** [comment]
Close for this tweak is deprecated.

**Pull request closed without merging**